bin/rubycritic

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env ruby
 
+# Always look in the lib directory of this gem
+# first when searching the load path
+$:.unshift File.expand_path("../../lib", __FILE__)
```

Comment:
I really can't wait for Ruby's shitty shellism's to die.
Let's replace 

```$:```

 with

```$LOAD_PATH```